variables.tf

```diff
@@ -2,6 +2,11 @@
 # Module Level Variables
 ##############################################################################
 
+variable "name" {
+  description = "Name for VPC"
```

Comment:
hmm, this might be confusing because this value will not actually be the vpc name, the vpc name will contain the prefix value too right?

Reply:
hmm, but `prefix` is a required variable right now (it has no default value). Should we default it to null?

Reply:
Looking more into it, looks like `prefix` is used throughout without a check for null so we should make it required